eth/gasprice/gasprice.go

```diff
@@ -215,6 +240,10 @@ func (oracle *Oracle) EstimateBaseFee(ctx context.Context) (*big.Int, error) {
 	}
 
 	baseFee = math.BigMin(baseFee, nextBaseFee)
+
+	if oracle.backend.ChainConfig().IsFeeConfigManager(new(big.Int).SetUint64(block.Header().Time)) {
```

Comment:
We already do this within `estimateNextBaseFee`: https://github.com/ava-labs/subnet-evm/blob/0d8ef8a7ed250a83ed34a3dc9baec165656cb4aa/consensus/dummy/dynamic_fees.go#L107

Reply:
yes but if we increase `minBaseFee` and `oracle.suggestDynamicFees(ctx)` reports a lower fee (since previous blocks has lower base fees) it will resolved to `nextBaseFee` in the 
https://github.com/ava-labs/subnet-evm/blob/0d8ef8a7ed250a83ed34a3dc9baec165656cb4aa/eth/gasprice/gasprice.go#L297
 which is `nextBaseFee` > minBaseFee.

This is not probably the ideal approach though. We should be changing `oracle.estimateNextBaseFee` to adjust this fee config changes.